extmod/moducryptolib.c

```diff
@@ -44,10 +44,21 @@ enum {
     UCRYPTOLIB_MODE_MIN = 0,
     UCRYPTOLIB_MODE_ECB,
     UCRYPTOLIB_MODE_CBC,
+#if MICROPY_PY_UCRYPTOLIB_CTR
+    UCRYPTOLIB_MODE_CTR = 6,
+#endif
     UCRYPTOLIB_MODE_MAX,
 };
 
+struct ctr_params {
+    // nonce is the IV of the AES context.
+
+    size_t nc_off;
+    uint8_t stream_block[16];
+};
+
 #if MICROPY_SSL_AXTLS
+
```

Comment:
This change contradicts https://github.com/micropython/micropython/wiki/ContributorGuidelines , "The same applies to unrelated changes - for example, don't put formatting changes and actual code changes in one commit."

Reply:
Moved to a separate formatting commit

Reply:
Will remove https://github.com/micropython/micropython/pull/4727#discussion_r280585799